src/Germany/Mews.Fiscalizations.Germany/V2/Client.cs

```diff
@@ -63,12 +64,19 @@ private Task<HttpResponseMessage> SendRequestAsync<TRequest>(HttpMethod method,
             var content = await response.Content.ReadAsStringAsync();
             if (response.IsSuccessStatusCode)
             {
-                return successFunc(JsonConvert.DeserializeObject<TDto>(content));
+                return Try.Catch<ResponseResult<TResult>, JsonReaderException>(_ => successFunc(JsonConvert.DeserializeObject<TDto>(content))).Get(
```

Comment:
Isn't it better to rather change the type of this method to return `Task<ITry<ResponseResult<TResult>, DeserializationError>>`, so that the usage is the place that decides how to display this error? 

We shouldn't really rely on the exception message to be meaningful. We have types for that.